independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "This is a connector to the Open Trivia Database to allows access to it's fun trivia questions",
+    "version": "1.0",
+     "contact": {
+      "name": "Kiveshan Naidoo, Rishay Sunder",
+      "email": "kiveshannaidoo9@gmail.com"
+    }
+  },
+  "host": "opentdb.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api_category.php": {
+      "get": {
+        "responses": {
+          "default": {

```

Comment:
@Kiveshan,, default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)